db/seeds.rb

```diff
@@ -3,5 +3,21 @@
 #
 # Examples:
 #
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+#  cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
+#  Mayor.create(name: 'Emanuel', city: cities.first)
+
+CSV.read("./seed_csvs/markets.csv").map do |row|
+  Market.create(id: row[0], name: row[1], address: row[2], city: row[3], county: row[4], state: row[5], zip: row[6])
+end
+
+CSV.read("./seed_csvs/vendors.csv").map do |row|
+  Vendor.create(id: row[0], name: row[1], num_employees:row[2], market_id:row[3])
+end
+
+CSV.read("./seed_csvs/products.csv").map do |row|
+  Product.create(id: row[0], name: row[1], vendor_id:row[2])
+end
+
+CSV.read("./seed_csvs/sales.csv").map do |row|
+  Sale.create(id: row[0], amount: row[1], purchase_time:row[2].to_datetime, vendor_id: row[3], product_id: row[4])
+end
```

Comment:
Have you tried creating new instances of your model classes after pushing this to Heroku? I know some people had issues with that related to the way the DB was seeded.

When you do, for example `Market.create(id: NUMBER, ...)`, SQLite knows that the next time we call `Market.create` without specifying and ID, it will assign a new ID that is greater than any of the IDs already in the database. However, Postgres is not quite so accommodating.

For Postgres you need to explicitly say what the next ID should be, otherwise it will try to give you and ID of 1 when you call `Market.create`, even though we already seeded a Market with that ID. If you have any trouble creating new markets or products, etc. on Heroku, I would recommend adding this to the end of your `seeds.rb` file:
```ruby
begin
  con = PG::Connection
  Market.connection.execute('SELECT setval(\'markets_id_seq\'::regclass, (SELECT MAX(id) FROM markets))')
  Vendor.connection.execute('SELECT setval(\'vendors_id_seq\'::regclass, (SELECT MAX(id) FROM vendors))')
  Product.connection.execute('SELECT setval(\'products_id_seq\'::regclass, (SELECT MAX(id) FROM products))')
  Sale.connection.execute('SELECT setval(\'sales_id_seq\'::regclass, (SELECT MAX(id) FROM sales))')
rescue NameError
end
```